### PR TITLE
LMB-457:  Disable mandatarissen that are in completed legislatuur

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

When a legislatuur is status completed it cannot be edited anymore. In the legislatuur all mandatarissen can also not be edited anymore as they are final. When going to a specific mandataris in the period all the edit buttons will be disabled.

## How to test

1. Change status of legislatuur to "Behandeld"
2. Check out the mandatarissen in the periode of the legislatuur
3. They should not be editable anymore

## Attachments

From Preparation

image

To behandeld

image

### CODE REVIEW

File: app/controllers/mandatarissen/persoon/mandataris.js [hunk 1]
```diff
@@ -67,4 +70,25 @@ export default class MandatarissenPersoonMandatarisController extends Controller
...
+    const behandeldeVergaderingen = await this.store.query(
+      'installatievergadering',
+      {
+        'filter[status][:uri:]': INSTALLATIVERGADERING_BEHANDELD_STATUS,
+        'filter[bestuursperiode][:id:]': bestuursperiode.id,
+      }
+    );
+
+    this.inCompletedLegislatuur = behandeldeVergaderingen.length >= 1;
+  });
```
Comment (JonasVanHoof) [inline app/controllers/mandatarissen/persoon/mandataris.js:93]: *

File: app/controllers/organen/index.js [hunk 2]
```diff
@@ -74,4 +76,17 @@ export default class OrganenIndexController extends Controller {
...
+    const behandeldeVergaderingen = await this.store.query(
+      'installatievergadering',
+      {
+        'filter[status][:uri:]': INSTALLATIVERGADERING_BEHANDELD_STATUS,
+        'filter[bestuursperiode][:id:]': this.model.selectedPeriod.id,
+      }
+    );
+
+    this.isInBehandeldLegislatuur = behandeldeVergaderingen.length >= 1;
+  }
```
Comment (JonasVanHoof) [inline app/controllers/organen/index.js:91]: *

File: app/routes/organen/orgaan/mandatarissen.js [hunk 3]
```diff
@@ -30,12 +31,22 @@ export default class OrganenMandatarissenRoute extends Route {
...
 
+    const behandeldeVergaderingen = await this.store.query(
+      'installatievergadering',
+      {
+        'filter[status][:uri:]': INSTALLATIVERGADERING_BEHANDELD_STATUS,
+        'filter[bestuursperiode][:id:]':
+          currentBestuursorgaan.get('bestuursperiode.id'),
+      }
+    );
+
```
Comment (JonasVanHoof) [inline app/routes/organen/orgaan/mandatarissen.js:42]: *

**Review by JonasVanHoof — COMMENTED** [review]
Places where I use the query

**Review by ruben-bottu — APPROVED** [review]

**Pull request merged**